src/swagger.json

```diff
@@ -5,28 +5,56 @@
 		"parameters": {},
 		"requestBodies": {},
 		"responses": {},
-		"schemas": {},
+		"schemas": {
+			"Scenario": {
+				"properties": {
+					"status": {
+						"type": "string",
+						"enum": [
+							"success",
+							"error"
+						]
+					},
+					"output": {
+						"type": "string"
+					}
+				},
+				"required": [
+					"status",
+					"output"
+				],
+				"type": "object",
+				"additionalProperties": false
+			}
+		},
 		"securitySchemes": {}
 	},
 	"info": {
-		"title": "ts-nodejs-template",
-		"version": "0.0.0",
-		"description": "A typescript service template for API services using nodejs/express",
+		"title": "@digicatapult/morello-api",
+		"version": "0.0.1",
```

Comment:
Should be generated from package.json?

Reply:
tsoa generates this file, to be fair I think I put it in .gitignore. This is something I would like to keep away from repo as routes and swagger will be generated as part of build process

Reply:
file has been removed and .gitignore should take care of this.